opengrok-indexer/src/main/java/org/opengrok/indexer/history/FileHistoryCache.java

```diff
@@ -463,15 +456,8 @@ public void store(History history, Repository repository)
                     list = new ArrayList<>();
                     map.put(s, list);
                 }
-                /*
-                 * We need to do deep copy in order to have different tags
```

Comment:
Could you please explain what was the use case?
```
We need to do deep copy in order to have different tags per each commit.
```

Reply:
I think the author of the original code forgot that the tags were reset in `doFileHistory()`. Assigning the tags to history entries used shared across files would lead to wrong results.